registry/mainnet/interchain/squid.tokenlist.json

```
@@ -4836,7 +4836,7 @@
       "iconUrls": {
         "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/pundiai.svg"
       },
-      "deploymentMessageId": "0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06-98",
+      "deploymentMessageId": "0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06-64",
```

```suggestion
      "deploymentMessageId": "0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06-3",
```
since the ContractCall event index is 3 in the tx
https://axelarscan.io/gmp/0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06-3

---

Hi, @milapsheth However, ben previously told me that the last digits represent the Position in Block.

---

@zakir998, apologies that was my mistake. It should be `-3` not `-64`. You can see in this [tx](https://axelarscan.io/gmp/0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06) If you click `see more` you will see the `messageId` field, which shows that the message-id should be `0x2b62e96d2047fe855d60d6e018be1de92a80b24e4924c9a1d24d0363c203fd06-3`

---

I understand. Thank you.